src/components/Shape/Circle.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ShapeProps } from './Shape';
+
+export const Circle: React.FC<ShapeProps> = () => (
```

Comment:
None of these actually accept the `ShapeProps` here since they don't even accept props.

You'd want to remove all the hardcoded attributes (fills, viewbox, width/height, etc) then spread the props onto the SVG. The point being that a given shape could be used in other contexts and that the colors are under the control of the design system tokens rather than being hardcoded and subject to drift.

🎩